Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageAdapterTest.cs

```diff
@@ -5,63 +5,286 @@
 using Microsoft.AppCenter.Utils.Files;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
+using SQLitePCL;
 using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Microsoft.AppCenter.Test.Storage
 {
     [TestClass]
     public class StorageAdapterTest
     {
+        private StorageAdapter _adapter;
+
+        // Const for storage data.
+        private const string StorageTestChannelName = "storageTestChannelName";
+        private const string TableName = "LogEntry";
+        private const string ColumnChannelName = "Channel";
+        private const string ColumnLogName = "Log";
+        private const string ColumnIdName = "Id";
+        private const string DatabasePath = "databaseAtRoot.db";
+
+        [TestInitialize]
```

Comment:
It would be useful to have a test case level cleanup method to avoid side-effects of manipulating storage.

Reply:
Ignore my previous comment.

Cleanup is in https://github.com/microsoft/appcenter-sdk-dotnet/pull/1246/files#diff-2368ae65b69493af7b1d82d1d642e7d5R275.